scripts/ci/run-jenkins.sh

```diff
@@ -87,6 +87,7 @@ if [[ ${CI_TAGS} == *'product-sdks-ios'* ]];
    then
 	   echo "DISABLE_ANDROID=1" > sdks/Make.config
 	   echo "DISABLE_WASM=1" >> sdks/Make.config
+	   echo "IGNORE_PACKAGE_LLVM=1" >> sdks/Make.config
 	   export device_test_suites="Mono.Runtime.Tests System.Core"
```

Comment:
This is still backwards.

Reply:
It seems to be the behavior I expect. When you export IGNORE_PACKAGE_LLVM, CI will download the package rather than trying to compile LLVM. That seems like the expected default.

Reply:
IGNORE_PACKAGE_LLVM means ignore the prepackaged (downloaded) version. Its meaning is also part of the sdk api and xamarin.ios depends on this.

Reply:
`IGNORE_PACKAGE_LLVM` is used by Android to signify you do not want to call `package-llvm-llvm*`, not the user way around. The meaning should be fixed on iOS for it to mean the same thing everywhere.

Reply:
So what do I do here?

Reply:
Change the code so that it does the download if IGNORE_PACKAGE_LLVM is _not_ set.